Dockerfile

```diff
@@ -8,14 +8,21 @@ WORKDIR /go/src/github.com/openfaas/faas-swarm
 COPY vendor     vendor
 COPY handlers	  handlers
 COPY types      types
+COPY version    version
+COPY .git       .git
```

Comment:
@johnmccabe is this needed? Is it avoided in the CLI due to the way we `COPY .`?

Reply:
In CLI we do `copy .  .`  I did not copy everything because it will may increase the size of  image

Reply:
Theres no real need to be selective surely? Its a multi-stage build so the resulting image is `scratch` + the binary.

Reply:
Sure .. I will change that/